docs/src/components/Example.js

```diff
@@ -32,20 +32,20 @@ const exportDefaultMaybe = ({ code }) =>
 
 export default Demo;`;
 
+const dedupeArray = <T>(arr: Array<T>): Array<T> => [...new Set(arr)];
+
 const handleCodeSandbox = async ({ code, title }) => {
   const gestaltComponents = Object.keys(gestalt);
   const additionalGestaltComponents = ['DatePicker'];
 
-  const usedComponents = [
```

Comment:
we are concatenating two deduped arrays. I think it makes more sense to dedupe _after_ concatenating two arrays.